tests/zino_test.py

```diff
@@ -36,14 +36,16 @@ def test_zino_should_not_crash_right_away(polldevs_conf_with_no_routers):
             str(seconds_to_run_for),
             "--polldevs",
             str(polldevs_conf_with_no_routers),
+            "--config-file",
+            str(zino_default_conf),
```

Comment:
Why do we need to add a config-file option to this test?  The `zino.toml` file is not required to run Zino, but a `polldevs.cf` is (or there would be nothing for Zino to do).

Since a valid `polldevs.cf` is needed to run, adding this option here is just to ensure the Zino process has access to a valid pollfile during the test, so it can run a "normal" startup routine.

Testing that Zino actually picks up the settings from a valid `zino.toml` would be best as a separate test.  Also, maybe a test for what Zino should do in the case of an invalid `zino.toml` file.

Reply:
What should Zino do in case of an invalid `zino.toml` file? Log an error and run anyways or stop running?

Reply:
as the zino.toml file will be parsed at server start time i think this i a good solution:
* In the event of non-existant zino.toml, then load all defaults and start the server. 
* if a zino.toml file exist and are parse-able parse all elements as overrides to the loaded defaults.
* on a existent but syntactli invalid zino.toml, i think the server should error out and not start with an "invalid" configuration.
also please look trough the config file for illegal options .. if this only should be printed as a warning or should be a hard error i've not concluded with at this point.. :)